saleor/graphql/discount/mutations/voucher/voucher_create.py

```diff
@@ -68,6 +84,14 @@ class VoucherInput(BaseInputObjectType):
     only_for_staff = graphene.Boolean(
         description="Voucher can be used only by staff user."
     )
+    single_use = graphene.Boolean(
+        description=(
+            "When set to 'True', each voucher is limited to a single use; "
+            "otherwise, usage remains unrestricted."
```

Comment:
> usage remains unrestricted

This is not clear to me. If the usage remains unrestricted, how `usage_limit` behave in that case?
So assuming, I have `single_use`:False, and `usageLimit`:5. The usage should be limited to 5, right?